services/src/projects/project.rs

```diff
@@ -659,49 +659,52 @@ mod tests {
 
     #[test]
     fn serialize_symbology() {
-        let symbology = Symbology::Vector(VectorSymbology::Point(PointSymbology {
-            radius: NumberParam::Static(1),
+        let symbology = Symbology::Point(PointSymbology {
+            radius: NumberParam::Static { value: 1 },
```

Comment:
Why do we need a dict for this?

Reply:
because I changed the serialization to tag with type and specified `value` as attributename for the payload.